src/context/TastStoreContextProvider.tsx

```diff
@@ -1,35 +1,64 @@
 /* eslint-disable @typescript-eslint/no-empty-function */
 import { createContext, useState } from 'react';
-import { TASKS_STORAGE_KEY } from '../constants/storage';
+import { TASKS_LIST_STORAGE_KEY, TASK_GROUPS_STORAGE_KEY } from '../constants/storage';
 import { createLocalStorageStore } from '../tools/sotrage';
 import { TaskItemType } from '../types/TaskItemType';
+import { TaskItemGroupType } from '../types/TaskItemGroupType';
 
 export type TaskStoreContextType = {
+  taskGroupList: TaskItemGroupType[];
```

Comment:
No, let's just change initial structure of the data - to be smthng like 
*added Array just for better readability
![image](https://github.com/macopul/Todo-App-React/assets/21220135/fb8b8952-eaa0-4cb7-a3fa-68b4adc79c44)

Reply:
done as I showed You last time